ospool.osg-htc.org/development/frontend-template.xml

```diff
@@ -142,6 +142,10 @@
             <file absfname="/opt/osg-flock/ospool-pilot/itb/pilot/advertise-userenv" type="run:singularity" after_entry="True" const="True" executable="True" prefix="NOPREFIX" untar="False" wrapper="False">
                <untar_options cond_attr="TRUE"/>
             </file>
+            <file absfname="/opt/osg-flock/ospool-pilot/itb/pilot/hello-world" after_entry="True" const="True" executable="True" period="0" prefix="HELLO_WORLD_" untar="False" wrapper="False">
```

Comment:
```suggestion
            <file absfname="/opt/osg-flock/ospool-pilot/itb/pilot/hello-world" after_entry="True" const="True" executable="True" period="0" prefix="NOPREFIX" untar="False" wrapper="False">
```

I don't think your `helloworld` attribute advertised in the `hello-world` script needs to have a prefix.